src/app/core/top-panel/top-panel.component.ts

```diff
@@ -127,6 +127,20 @@ export class TopPanelComponent implements OnInit {
     return this.isWorkspaceSet() && !this.searchBoxHidden;
   }
 
+  public goToOrganization(page: string) {
+    if (this.workspace && this.workspace.organizationCode) {
+      this.store.dispatch(new RouterAction.Go({path: ['organization', this.workspace.organizationCode, page]}));
```

Comment:
`RouterAction.Go` is defined there only to make it easier to navigate to some other place when you need to do so in the effect classes. It is not necessary to do so in the components. Just more code is executed in this case.